src/data/team.js

```diff
@@ -357,6 +357,19 @@ Various design podcasts including [SimpleBits](https://show.simplebits.com/), [H
     alumni: true,
     color: 'orange'
   },
+  {
+    name: 'Shawn Allen',
+    title: 'Systems Designer',
+    handle: 'shawnbot',
+    cssFeature: 'box-sizing: border-box',
+    questionOne: `I worked in data visualization and map-making for most of my career, and solving design problems with data is my jam. To me there's something uniquely exciting about making a thing that can take complicated and/or unexpected input and, with no human intervention, reduce it to something visually comprehensible—and hopefully interesting!`,
+    questionTwo: `Anyone who has ever contributed to [A List Apart](http://alistapart.com/) or posted to the [webdesign-l](http://www.webdesign-l.com/) mailing list between 1999 and 2005. The [CSS Zen Garden](http://www.csszengarden.com/) was hugely inspirational for me, along with web standards pioneers like Eric Meyer, Mark Pilgrim, and Jeff Zeldman. [Jina](https://twitter.com/jina) is one of the hardest-working people in design systems, and her leadership in the community is a tremendous inspiration. I learned everything I know about SVG from [Sara Soueidan](https://twitter.com/SaraSoueidan), and everything I could _hope_ to know about CSS Grid from [Rachel Andrew](https://twitter.com/rachelandrew) and [Jen Simmons](https://twitter.com/jensimmons). If you want to learn about web components, follow [Monica Dinculescu](https://twitter.com/notwaldorf) and [Mu-An Chiou](https://twitter.com/muanchiou).`,
+    favoriteTools: 'Vim, Things, Chrome DevTools, curl',
+    avatar: 'https://user-images.githubusercontent.com/586552/46751266-f81b7380-cc87-11e8-8459-2166a1c40f63.jpg',
+    gif: 'https://camo.githubusercontent.com/ab7c209f3da3ff14493ff129778589612df7cd8d8ec9b885ca2ed8619bd721a1/68747470733a2f2f692e67697068792e636f6d2f6d656469612f6b484f6c7a4c774a44566f4e572f67697068792e77656270',
+    alumni: true,
```

Comment:
```suggestion
    alumni: false,
```

Come back Shawn 🥲

Reply:
We miss you Shawn! 

![miss you shawn](https://user-images.githubusercontent.com/10384315/183522294-47d1fea5-0be9-45ee-a2d7-e9454dea922f.gif)